packages/scripts/src/monitoring-dvm2.0/unstake.ts

```diff
@@ -48,9 +48,10 @@ async function main() {
       `VotingV2 balance should be between 0 and ${numberSlashedEvents} but is ${votingV2BalanceWithoutExternalTransfers}`
     );
 
-  // VotingV2 cumulativeStake should be 0
+  // VotingV2 cumulativeStake should be 0 with a tolerance of numberSlashedEvents, as every slash can result in 1 WEI of
+  // imprecision due to rounding.
   const cumulativeStake = await votingV2.cumulativeStake();
-  if (cumulativeStake.toString() != "0") throw new Error("VotingV2 cumulativeStake is not 0");
+  if (cumulativeStake.gt(numberSlashedEvents)) throw new Error("VotingV2 cumulativeStake is not 0");
```

Comment:
maybe adjust error message to include expected range, similar to votingV2 Balance check above